scripts/platforms/linkedin_apply.py

```
@@ -0,0 +1,463 @@
+                    f"\nQUESTION -> {question}"
+                )
+
+                answer = generate_answer(question)
+
+                await textarea.fill(answer)
+
+                print(
+                    "Answer inserted."
+                )
+
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)